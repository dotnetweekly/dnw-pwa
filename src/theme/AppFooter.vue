<template>
  <footer class="footer">
    <div class="container">
      <div class="content">
        <div class="footer-menu has-text-left is-pulled-left">
          <router-link to="about">About</router-link>
          <router-link to="about">Advertise</router-link>
          <router-link to="about">Code of Conduct</router-link>
        </div>
        <div class="has-text-right is-pulled-right">
          Follow us on
          <a href="https://twitter.com/dotnetweekly" target="_blank">dotNET Weekly</a>
        </div>
      </div>
    </div>
  </footer>
</template>
<style lang="scss" scoped>
@import "../_variables";

.footer {
  padding: $is-size-4 $is-size-6;
}
.footer-menu a {
  padding-right: $is-size-4;
}
</style>
