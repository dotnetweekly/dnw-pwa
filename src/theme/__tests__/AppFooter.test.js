import { shallow } from "vue-test-utils";
import AppFooter from "../AppFooter.vue";

describe("AppFooter.test.js", () => {
  let cmp;

  beforeEach(() => {
    cmp = shallow(AppFooter, {});
  });

  it("has the expected html structure", () => {
    expect(cmp.element).toMatchSnapshot();
  });
});
