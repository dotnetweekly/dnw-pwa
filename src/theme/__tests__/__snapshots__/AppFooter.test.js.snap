// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter.test.js has the expected html structure 1`] = `
<footer
  class="footer"
>
  <div
    class="container"
  >
    <div
      class="content"
    >
      <div
        class="footer-menu has-text-left is-pulled-left"
      >
        <router-link
          to="/newsletters"
        >
          Newsletters
        </router-link>
         
        <router-link
          to="/about"
        >
          About
        </router-link>
      </div>
       
      <div
        class="has-text-right is-pulled-right"
      >
        <a
          href="https://twitter.com/dotnetweekly"
          rel="noopener"
          target="_blank"
        >
          <i
            aria-hidden="true"
            class="icon-twitter"
          />
           Follow Us
        
        </a>
      </div>
       
      <div
        class="is-clearfix"
      />
    </div>
  </div>
</footer>
`;
